.pre-commit-config.yaml

```diff
@@ -10,15 +10,15 @@ repos:
     hooks:
     -   id: check-toml
     -   id: check-yaml
--   repo: https://github.com/psf/black
-    rev: 24.8.0
-    hooks:
-    -   id: black
 -   repo: https://github.com/astral-sh/ruff-pre-commit
```

Comment:
wasn't ruff enabled in the previous commit? Do you know why it did not it apply isort previously?

Reply:
yes, but the previous PR only enabled the `F` ruleset, see https://github.com/Consensys/mythril/blob/develop/pyproject.toml#L7 ... to find and remove unused code/imports.

ruff has many more rules that can be activated, see https://docs.astral.sh/ruff/rules/ 

This PR enables isort by adding the `I` to the activated rules, see https://github.com/dbast/mythril/blob/isort/pyproject.toml#L7

Reply:
I mean, I don't see that in this PR. Is the `I` in a different PR?

Reply:
nvm, found it,
Github compressed some files for me